package.json

```diff
@@ -59,6 +59,8 @@
     "typescript": "^4.7.3"
   },
   "dependencies": {
+    "ajv": "^8.11.0",
+    "ajv-keywords": "^5.1.0",
```

Comment:
I was about to ask if this needed to be a dev dependency, but no, if we are using it to parse and/or validate, it needs to be included in prod dist.

Reply:
Correct!  They are runtime dependencies (companions to the typescript dev dependency).